docs/guides/src/main/getting-started/templates/first-app.adoc

```diff
@@ -1,36 +1,39 @@
-== Secure your first app
+== Secure the first application
 
-Let's try to secure our first application. First step is to register this application with your Keycloak instance:
+To secure the first application, you start by registering the application with your Keycloak instance:
 
-. Open the {links-admin-console}
-. Click 'Clients'
-. Click 'Create client'
+. Open the {links-admin-console}.
+. Click *Clients*.
+. Click *Create client*
 . Fill in the form with the following values:
-** Client type: `OpenID Connect`
-** Client ID: `myclient`
-. Click 'Next'
-. Make sure 'Standard flow' is enabled
-. Click 'Save'
+** *Client type*: `OpenID Connect`
+** *Client ID*: `myclient`
+. Click *Next*
+. Confirm that *Standard flow* is enabled.
+. Click *Save*.
 
 image::add-client-1.png[Add Client]
 
-After the client is created you need to update the following values for the client:
+After the client is created, make these updates to the client:
 
-. Valid redirect URIs: `https://www.keycloak.org/app/*`
-. Web origins: `https://www.keycloak.org`
-
-Remember to click `Save`.
+. Scroll down to *Access settings*.
+* Set *Valid redirect URIs* to `https://www.keycloak.org/app/*`
+* Set *Web origins* to `https://www.keycloak.org`
```

Comment:
Those links are data that should be entered, and not clicked on. Therefore I added a passthrough so they are rendered as text and not as links. This was this way in the original docs, still it should be changed.

```suggestion
* Set *Valid redirect URIs* to `+https://www.keycloak.org/app/*+`
* Set *Web origins* to `+https://www.keycloak.org+`
```